cvmfs/sync_mediator.h

```diff
@@ -50,6 +51,49 @@ struct Counters;
 
 namespace publish {
 
+class SyncDiffReporter : public DiffListener {
+ public:
+  explicit SyncDiffReporter(bool print_dots,
+                            unsigned int processing_dot_interval = 100)
+      : print_dots_(print_dots),
+        processing_dot_interval_(processing_dot_interval),
```

Comment:
Having the interval configurable is a nice idea!  As we don't make use of it for the time being, let's remove the argument from the constructor (but let's keep the member, initialized with 100).